Documentation/DE/Referenz/Hooks/Index.rst

```diff
@@ -549,3 +550,116 @@ It's used like this:
         * @return void
         */
           public function modifyCancelEmail(Tx_Seminars_Model_Registration $registration, Tx_Oelib_Mail $eMail) {…}
+
+.. _backendregistrationlistview_de:
+
+Hooks zur Backend-Registrierungsliste
+"""""""""""""""""""""""""""""""""""""
+
+Es gibt 3 Hooks in die Backend-Registrierungsliste. Die Hooks werden während der Erstellung der
+Backend-Registrierungsliste aufgerufen:
+
+* Bevor der Tabellenkopf in HTML umgewandelt wird
+* Bevor eine Tabellenzeile zu einer Registrierung in HTML umgewandelt wird
+* Bevor der Tabellenfuß in HTML umgewandelt wird
```

Comment:
As these are no full sentences, please lowercase the first word of each list item.